### ISSUE TITLE
Accessibility: Change Careers block links to open in the same tab/window

### ISSUE THREAD

**Issue by wrenaria** [UCDavisLibrary/ucdlib-theme-wp#326]
Currently the links in the [Careers](https://library.ucdavis.edu/careers/) block open a new tab. Unless I missed it, I didn't see a setting for this on the block settings, so I am assuming this behavior is hard-coded into the block.

All links used in these blocks should be changed to open in the **same tab/window** to meet [accessibility standards for link behavior](https://www.w3.org/WAI/WCAG22/Techniques/general/G200).

**Comment by spelkey-ucd** [comment]
Released [v4.4.0](https://github.com/UCDavisLibrary/ucdlib-theme-wp/releases/tag/v4.4.0)
